source/css/style.css

```
@@ -122,5 +122,64 @@ list-style: none;
 .main-container {
   max-width: var(--max-content-width);
   margin: 0 auto;
-  padding: 50px;
+  padding-top: 30px;
+}
+
+.populars-grid {
+  list-style: none;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```

Можно написать еще изящней)
```suggestion
  grid-template-columns: repeat(3, 1fr);
```

---

Да я читал. Насколько это даёт плюсов в карму? Ведь сути то не меняет =))))))

---

Сути не меняет, конечно, но позволяет помнить про такою запись. Ну и покажет твое знание на собесе, в тестовом, например)